internal/proto/codec/builtin.go

```diff
@@ -728,8 +728,7 @@ func DecodeError(msg *proto.ClientMessage) *hzerrors.ServerError {
 		return nil
 	}
 	holder := errorHolders[0]
-	err := hzerrors.NewServerErrorImpl(holder.ErrorCode(), holder.ClassName(), holder.Message(), holder.StackTraceElements(), 0, "")
-	return &err
+	return ihzerrors.NewServerError(holder.ErrorCode, holder.ClassName, holder.Message, holder.StackTraceElements, 0, "")
```

Comment:
Why do we have hardcoded parameters? `0` and `""`

Reply:
Removed them at: https://github.com/hazelcast/hazelcast-go-client/pull/586/commits/1e6b1575cd47beb31fae6ee74e469a5ca394da3c